personalize.module

```diff
@@ -3195,66 +3195,3 @@ function _personalize_is_action_used($action_name) {
 
   return $query->count()->execute() > 0;
 }
-
```

Comment:
Removed personalize_form_element_path_validate() because it is moved to visitor_actions and renamed to visitor_actions_form_element_path_validate().